src/Native/LibTorchSharp/THSTensor.cpp

```diff
@@ -1568,6 +1578,19 @@ Tensor THSTensor_row_stack(const Tensor* tensors, const int length)
     CATCH_TENSOR(torch::row_stack(toTensors<at::Tensor>((torch::Tensor**)tensors, length)));
 }
 
+void THSTensor_meshgrid(const Tensor* tensors, const int64_t length, const char* indexing, Tensor* (*allocator)(size_t length))
+{
+    std::string str = indexing;
+    at::TensorList tList = toTensors<at::Tensor>((torch::Tensor**)tensors, length);
+    CATCH(
+        auto res = torch::meshgrid(tList, indexing);
+        const size_t sz = res.size();
+        Tensor * result = allocator(sz);
```

Comment:
>result [](http://example.com/codeflow?start=17&length=6)

Q:
I assume some allocations occur here. are these allocation internally tracked by the tensors and get freed at some point?

Reply:
Yes, they are tracked on the managed side.